crates/amaru/Cargo.toml

```diff
@@ -46,6 +46,7 @@ opentelemetry-otlp = { version = "0.27.0", features = [
     "http-proto",
     "reqwest-client",
 ] }
+rand = "0.8"
```

Comment:
Is `rand` only used for tests by chance?

Reply:
Good question. I think it was in `dev-dependencies` and I moved it there because I moved some code back and forth between tests and prod.